ClientApp/viking-vault/src/AdminDashboard/ActivatePaymentToggle.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import {constants} from "../Resources/Constants";
+import './toggle.css';
+
+const API_URL = `${constants.baseUrl}`;
+
+interface IPaymentToggleProps{
+  paymentId: Number
+}
+
+class ActivatePaymentToggle extends React.Component<IPaymentToggleProps, any>{
+  constructor(props: IPaymentToggleProps){
+    super(props)
+    this.handleCheckBox = this.handleCheckBox.bind(this)
+    this.state = {
+      checked: false
+    }
+  }  
+
+  handleCheckBox(e:any){
+    this.setState({
+      checked: e.target.checked
+    })
+    fetch(API_URL+"PaymentToggling/"+this.props.paymentId, {
+      method: "PUT",
+      headers: {
+      'Accept': 'application/json',
+      'Content-Type': 'application/json'    
+      },
+      body: this.state.checked
+    })
+  }
+
+  componentDidMount(){
+    fetch(API_URL+"PaymentToggling/"+this.props.paymentId, {
+      method: "GET",
+      headers: {
+      'Accept': 'application/json',
+      'Content-Type': 'application/json'
+      }
+    })
+      .then(response => {
+        if( response.status === 500)
+        {
```

Comment:
👮‍♂️🚔JavaScript police violation: did not used the convention to write the curly on the same line
Pay your fine by writing code blocks like this! Thank you, JavaScript Police Department (JSPD)
blah {
}